app/src/main/java/com/asfoundation/wallet/onboarding_new_payment/payment_methods/OnboardingPaymentMethodsFragment.kt

```diff
@@ -62,8 +63,13 @@ class OnboardingPaymentMethodsFragment : BasePageViewFragment(),
         args = OnboardingPaymentMethodsFragmentArgs.fromBundle(requireArguments())
         handlePaymentMethodList()
         setStringWithLinks()
+        viewModel.fetchWalletInfo()
         viewModel.collectStateAndEvents(lifecycle, viewLifecycleOwner.lifecycleScope)
-        ChallengeRewardManager.create(this.requireActivity())
+    }
+
+    override fun onResume() {
```

Comment:
on the first payment should fyber wall be shown?

Reply:
Better to put the code in onCreate

Reply:
> on the first payment should fyber wall be shown?

I added the Offer Wall there because i saw other payments going there as well, and i didn't know what flow first payment is ':D But i can take Offer Wall out of there